tests/passes/writeset_underapproximation_test.py

```diff
@@ -9,8 +9,6 @@
 M = dace.symbol("M")
 K = dace.symbol("K")
 
-pipeline = Pipeline([UnderapproximateWrites()])
```

Comment:
the pipeline in the tests is likely completely unnecessary (as it's a pipeline of size 1). I can ignore that for this PR but it serves no purpose.

Reply:
Forget about this comment, it's necessary because of pass dependencies.